core/chaincode/platforms/golang/hash.go

```diff
@@ -138,7 +135,11 @@ func getCodeFromHTTP(path string) (codegopath string, err error) {
 	//     . more secure
 	//     . as we are not downloading OBC, private, password-protected OBC repo's become non-issue
 
-	env[gopathenvIndex] = "GOPATH=" + codegopath + ":" + origgopath
+	os.Setenv("GOPATH", codegopath+":"+origgopath)
+	// Get a copy of that new env for the go get command
+	env := os.Environ()
+	// and reset GOPATH to its original value
+	os.Setenv("GOPATH", origgopath)
```

Comment:
I understand. The result is the same though. I'm merely changing the env temporarily to get a copy ready for the exec.Command. Line142 resets the gopath so nothing has changed.

But this way I can take advantage of os.Getenv and os.Setenv rather than doing it the hard way with string manipulations. :-)